aiida/backends/control.py

```diff
@@ -0,0 +1,80 @@
+# -*- coding: utf-8 -*-
+###########################################################################
+# Copyright (c), The AiiDA team. All rights reserved.                     #
+# This file is part of the AiiDA code.                                    #
+#                                                                         #
+# The code is hosted on GitHub at https://github.com/aiidateam/aiida-core #
+# For further information on the license, see the LICENSE.txt file        #
+# For further information please visit http://www.aiida.net               #
+###########################################################################
+"""Module for overall repository control commands."""
+# Note: these functions are not methods of `AbstractRepositoryBackend` because they need access to the orm.
+# This is because they have to go through all the nodes to gather the list of keys that AiiDA is keeping
+# track of (since they are descentralized in each node entry).
+# See the get_unreferenced_keyset function
+from typing import Optional, Set
+
+from aiida.common.log import AIIDA_LOGGER
+from aiida.manage.manager import get_manager
+from aiida.orm.implementation import Backend
+
+__all__ = ('MAINTAIN_LOGGER',)
+
+MAINTAIN_LOGGER = AIIDA_LOGGER.getChild('maintain')
```

Comment:
Mmm but then the question is how do I manage the inheritance of loggers? Because here I just inherit from the `AIIDA_LOGGER` so I can just do:

```python
MAINTAIN_LOGGER = AIIDA_LOGGER.getChild('backend.control')
```

But then any other logger that inherits from this one will have `aiida.backend.control` prefix even though these are in `aiida.repository.backend.xxxxxx`...

Reply:
Ok, let's leave this for now.